src/lib/ng-package/entry-point/write-package.transform.ts

```diff
@@ -10,6 +10,7 @@ import * as log from '../../utils/log';
 import { globFiles } from '../../utils/glob';
 import { EntryPointNode, isEntryPointInProgress, isPackage, PackageNode, fileUrl } from '../nodes';
 import { Node } from '../../graph/node';
+import { CompilationMode } from '@angular/compiler-cli/src/ngtsc/transform';
```

Comment:
I'd avoid this import and just stick to `'partial'|'full'` as that's how the option is defined. There is no guarantee that the internal enum has those exact values; in fact I don't expect this to work as the enum is numeric:

https://github.com/angular/angular/blob/3a823abcc554cffdda1e14e181dadad9063701de/packages/compiler-cli/src/ngtsc/transform/src/api.ts#L22-L32